cli/cli/Services/BeamoLocalSystem_HttpMicroservice.cs

```diff
@@ -34,6 +34,11 @@ public partial class BeamoLocalSystem
 			BeamoProtocolType.HttpMicroservice,
 			async (definition, protocol) =>
 			{
+				if (string.IsNullOrEmpty(buildContexPath))
+				{
+					buildContexPath = _configService.GetDockerBuildContextPath();

```

Comment:
remove this parameter and just get the value from this method from ConfigService